src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -545,6 +545,14 @@ impl ThinPool {
                 self.resume()?;
                 self.set_extend_state(data_extend_failed, meta_extend_failed);
             }
+            ThinPoolStatus::Error => {
+                // Do not set the state, because it is unknown.
+                warn!(
+                    "Devicemapper could not obtain the status for devicemapper thinpool device {} belonging to pool with UUID {}",
+                    self.thin_pool.device(),
+                    pool_uuid
+                );
```

Comment:
I believe adding the pool name would be useful for this message too.

Reply:
Pool name is not available here.